src/test/fuzz/locale.cpp

```diff
@@ -82,10 +82,8 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     assert(atoi64c_without_locale == atoi64c_with_locale);
     const int atoi_with_locale = atoi(random_string);
     assert(atoi_without_locale == atoi_with_locale);
-    const std::string i64tostr_with_locale = i64tostr(random_int64);
-    assert(i64tostr_without_locale == i64tostr_with_locale);
-    const std::string itostr_with_locale = itostr(random_int32);
-    assert(itostr_without_locale == itostr_with_locale);
+    const std::string tostring_with_locale = ToString(random_int64);
+    assert(tostring_without_locale == tostring_with_locale);
```

Comment:
Isn't this comparing a string with itself now?

Reply:
```cpp
assert(tostring_without_locale
    == tostring_with_locale);

Reply:
The fuzzer's goal is to assert the function does not respond to changing the locale. So generating the same string with two different locales is needed. The check should assert that they are the same.